topics/ajax/index.md

```diff
@@ -0,0 +1,9 @@
+---
+display_name: AJAX
+short_description: Ajax is a technique for creating better, faster, and more interactive web applications.
```

Comment:
I don't think "better" and "faster" belong here (arguably opinionated).

This would be perfect IMO: Ajax is a technique for creating interactive web applications.